eng/common-variables.yml

```diff
@@ -18,7 +18,9 @@ variables:
     - group: DotNet-HelixApi-Access
     - group: SDL_Settings
     - name: _InternalBuildArgs
-      value: /p:DotNetSignType=$(_SignType) 
+      value: /p:DotNetSignType=$(_SignType)
         /p:TeamName=$(_TeamName)
         /p:DotNetPublishUsingPipelines=true
         /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+        /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
+        /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)

```

Comment:
hmm I don't think this should be necessary, or at least we don't do that in other pipelines from what I can see

Reply:
In the official pipeline we collect PDBs as build artifacts, shouldn't this effort make sure we produce symbols NuGet package instead?